sphinx_gherkindoc/cli.py

```diff
@@ -207,6 +208,13 @@ def main() -> None:
         help="Include steps glossary under the given name."
         " If not specified, no glossary will be created.",
     )
+    parser.add_argument(
+        "-T",
+        "--group-step-glossary",
+        action="store_true",
+        default=False,
+        help="Group step glossary by step type",
+    )
```

Comment:
If we ever want to group by other mechanisms this might be obsolete.
I was going to suggest changing this to a choice list, but since I can't think of how else I would group it, that would be overkill.
Just jotting down thoughts, no change requested.